src/NzbDrone.Core/Indexers/HttpIndexerBase.cs

```diff
@@ -109,6 +108,11 @@ public override IList<ReleaseInfo> Fetch(SpecialEpisodeSearchCriteria searchCrit
             return FetchReleases(g => g.GetSearchRequests(searchCriteria));
         }
 
+        public override HttpRequest GetDownloadRequest(Uri link)
```

Comment:
Instead of a URI can this accept a string? Looks like the two callers of this accept a string and then have to convert to a URI, when that could be done here instead.

Reply:
It doesn't look that applies here, plus as-is it's a string converted to a URI then back to a string when `link.AbsoluteUri` is used in `HttpIndexerBase`. If it is invalid then that should be dealt with by `HttpUri`